ext/node/polyfills/path/common.ts

```diff
@@ -18,15 +25,19 @@ import { SEP } from "ext:deno_node/path/separator.ts";
  * ```
  */
 export function common(paths: string[], sep = SEP): string {
-  const [first = "", ...remaining] = paths;
+  const [first = "", ...remaining] = ArrayFrom(paths);
```

Comment:
Please use `SafeArrayIterator` from primordials.

```suggestion
  const [first = "", ...remaining] = new SafeArrayIterator(paths);
```